packages/documentation/src/stories/foundation/color/color.styles.scss

```diff
@@ -0,0 +1,76 @@
+@use '@swisspost/design-system-styles/core' as post;
+
+.color-square {
+  border: 2px solid var(--post-contrast-color);
+}
+
+.color-square__inner {
+  aspect-ratio: 4/1;
+  padding: 0.75rem;
+  display: flex;
+  align-items: center;
+
+  h3 {
+    margin: 0;
+  }
+}
+
+.color-square__content {
+  border-top: 2px solid var(--post-contrast-color);
+  padding: 0.75rem;
+}
+
+.color-square__section {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(15rem, 1fr));
+  gap: 1.5rem;
+}
+
+h1 {
+  margin-block: 5rem 1rem;
+
+  &:first-of-type {
+    margin-block-start: 0;
+  }
+}
+
+h2 {
+  margin-block: 3rem 1rem;
+}
+
+h3 {
+  text-transform: capitalize;
+}
+
+body {
+  @include post.font-curve(regular);
+  font-weight: 300;
+}
+
+dl {
+  display: grid;
+  grid-template-columns: max-content 1fr;
+  gap: 0.5rem 1rem;
+}
+
+dl,
+dt,
+dd {
+  @include post.font-curve(small);
+  margin: 0;
+  padding: 0;
+}
+
+code {
+  font-family: monospace;
+  display: inline-block;
+  font-size: 0.8em;
+  line-height: 1;
+  margin: 0 0.125rem;
+  padding: 0.1875rem 0.3125rem;
+  white-space: nowrap;
+  border-radius: 0.1875rem;
+  border: 1px solid var(--post-gray-20);
+  color: var(--post-gray-80);
+  background-color: var(--post-light);
+}
```

Comment:
Please include this in the preview-head.html file and replace the `#docs-root code` if it is present in your branch